### PR TITLE
Improvements on added UserManager toggles

### PR DESCRIPTION
**Pull request by quh4gko8**

### CODE REVIEW

**Comment by quh4gko8** [comment]
has been tested on emulator and found to have same logs as if it's logged out from added logout option

the said logs refer to [this one](https://github.com/GrapheneOS/os-issue-tracker/issues/258#issuecomment-684677223)

**Comment by muhomorr** [comment]
Need to factor out code from `onPreferenceChange()`, it's the same as `DISALLOW_INSTALL_APPS` and `DISALLOW_INSTALL_UNKNOWN_SOURCES` restrictions.

File: src/com/android/settings/users/UserDetailsSettings.java [hunk 1]
```diff
@@ -533,4 +533,25 @@ private boolean shouldShowSetupPromptDialog() {
         //  return true so there will be no setup prompt dialog shown to the user anymore.
         return isSecondaryUser(mUserInfo) && !mUserInfo.isInitialized();
     }
+
+    private void setUserRestriction(String userManagerKey, boolean enabled) {
+        if (mUserInfo.isGuest()) {
+            mDefaultGuestRestrictions.putBoolean(userManagerKey, !enabled);
```
Comment (muhomorr) [inline src/com/android/settings/users/UserDetailsSettings.java]: I think negation should be done in the callers of this method, not in the method itself.

**Comment by quh4gko8** [comment]
This PR now also changes the description to match AOSP guidelines to add settings, as well as to expose "allow to install apps" toggle to Guest users, where it's allowed by default in AOSP.

File: res/values/strings.xml [hunk 2]
```diff
@@ -8065,9 +8065,11 @@
     <!-- Title of preference to enable calling and SMS [CHAR LIMIT=45] -->
     <string name="user_enable_calling_sms">Turn on phone calls &amp; SMS</string>
     <!-- Title of preference to disallow installing apps [CHAR LIMIT=45] -->
-    <string name="user_disallow_install_apps">Disallow installing apps</string>
+    <string name="user_disallow_install_apps">Allow installing apps</string>
```
Comment (muhomorr) [inline res/values/strings.xml:8068]: String ids have to be updated too (ie `user_allow_install_apps`)

File: res/values/strings.xml [hunk 3]
```diff
@@ -8065,9 +8065,11 @@
...
     <!-- Title of preference to disallow installing apps [CHAR LIMIT=45] -->
-    <string name="user_disallow_install_apps_unknown_sources">Disallow installing apps from third party sources</string>
+    <string name="user_disallow_install_apps_unknown_sources">Allow installing apps from third party sources</string>
+    <!-- Title of preference to disallow user running from background [CHAR LIMIT=45] -->
+    <string name="user_disallow_run_in_background">Allow running on background</string>
```
Comment (muhomorr) [inline res/values/strings.xml:8072]: "on background" -> "in background"

**Review by muhomorr — CHANGES_REQUESTED** [review]

**Review by muhomorr — APPROVED** [review]

**Pull request merged**